test/test.js

```diff
@@ -3,17 +3,35 @@ import 'babel-polyfill';
 
 import assert from 'assert';
 import { exec } from 'mz/child_process';
-import { exists, readFile, writeFile } from 'mz/fs';
+import { exists, readFile, writeFile, mkdtemp, mkdir } from 'mz/fs';
+import { join, sep, normalize } from 'path';
+
 
 import getFilesUnderPath from '../src/util/getFilesUnderPath';
 
 let originalCwd = process.cwd();
 
+async function mkdTempSafe (prefix) {
```

Comment:
Can you add a comment explaining what `safe` means here? Looks like it's doing the equivalent of `mkdir -p`? You could probably also skip this by making sure the `tmp-projects` folder always exists. I think one way people do that is to commit an empty `.gitkeep` file in the directory (since git doesn't allow empty tracked directories).